build/releaseBuild.yml

```diff
@@ -0,0 +1,223 @@
+trigger: none
+pr: none
+
+name: $(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+# Create a daily midnight build for release builds on master to ensure our release builds function
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - dev
+
+variables:
+  - group: Middleware
+  - name: BuildPlatform
+    value: 'any cpu'
+  - name: BuildConfiguration
+    value: 'Release'
+
+jobs:
+- job: build
+  pool:
+    name: MwWilson1EsHostedPool
+    demands:
+    - msbuild
+    - DotNetFramework
+    - visualstudio
+  timeoutInMinutes: 360
+
+  steps:
+  - task: UseDotNet@2
```

Comment:
Out of curiosity: Why do we need two use .Net Core sdk 2.1? I see one as .818 and the other as .x.

Reply:
I was getting some build issues on one of the tests, requiring .net core sdk 2.x, and it wasn't picking up the .818 version. It still uses the .818 version but it's now picked up by those tests.

Reply:
SDK 2.x is required by the compliance tooling.